spyder/plugins/editor/widgets/editorstack/editorstack.py

```diff
@@ -355,6 +355,10 @@ def __init__(self, parent, actions, use_switcher=True):
         self.create_new_file_if_empty = True
         self.indent_guides = False
         self.__file_status_flag = False
+        self.mouse_shortcuts = {'jump_to_position': 'Alt',
+                                'goto_definition': 'Ctrl',
+                                'add_remove_cursor': 'Ctrl+Alt',
+                                'column_cursor': 'Ctrl+Alt+Shift'}
```

Comment:
```suggestion
        self.mouse_shortcuts = {
            'jump_to_position': 'Alt',
            'goto_definition': 'Ctrl',
            'add_remove_cursor': 'Ctrl+Alt',
            'column_cursor': 'Ctrl+Alt+Shift'
        }
```

Fix Black formatting.